ecs-cli/modules/utils/compose/convert_task_definition.go

```diff
@@ -775,19 +780,21 @@ func ConvertToVolumes(volumeConfigs map[string]*config.VolumeConfig) *volumes {
 	if volumeConfigs != nil {
 		for name, config := range volumeConfigs {
 			if config != nil {
+				// NOTE: If Driver field is not empty, this
+				// will add a prefix to the named volume on the container
 				if config.Driver != "" {
-					log.WithFields(log.Fields{
-						"volume name": name,
-						"option name": "driver",
-					}).Warn("Skipping unsupported YAML option...")
+					return nil, errors.New("Volume driver is not supported")
 				}
+				// Driver Options must relate to a specific volume driver
 				if len(config.DriverOpts) != 0 {
-					log.WithFields(log.Fields{
-						"volume name": name,
-						"option name": "driver_opts",
-					}).Warn("Skipping unsupported YAML option...")
+					return nil, errors.New("Volume driver options is not supported")
 				}
+				// NOTE: explicitly setting external to false
+				// will cause compose up to fail, but if not
+				// spcified false will be set as default empty
+				// value.
 				if config.External.External {
+					return nil, errors.New("External option is not supported")
 					log.WithFields(log.Fields{
```

Comment:
I think the warning should just be removed since we are not skipping this option, we are failing on it. Also, since it comes after a `return` this statement will not run anyway...